masterlist.yaml

```diff
@@ -9774,3 +9774,54 @@ plugins:
         util: 'SSEEdit v4.0.1n RC4 for 4.0.2'
       - crc: 0xDA25CBFC
         util: 'SSEEdit v4.0.1n RC4 for 4.0.2'
+  - name: 'MissingEZs_AllExteriors.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/23609'
+        name: 'Missing Encounter Zones Fixed on Nexus Mods'
+    group: *highPriorityGroup
```

Comment:
If you're setting a group, then you'll need to know if it should load after mods or before mods in that group.
It's to avoid sorting errors.

Enhanced Lighting for ENB (ELE)
ELFXEnhancer
ELFX - Hardcore
Luminosity - Lightweight Lighting
Luminosity Lighting Overhaul
Alternate Start - Live Another Life
Open Cities Skyrim
Immersive Citizens - AI Overhaul

Reply:
Enhanced Lighting for ENB (ELE)
![image](https://user-images.githubusercontent.com/1944639/53516838-2843a180-3ac5-11e9-988d-3f75c852e15f.png)

Reply:
One conflict with Open Cities
![image](https://user-images.githubusercontent.com/1944639/53518123-4232b380-3ac8-11e9-96f8-9ac84f08d99c.png)

Reply:
Yes, it should load before ELE and ELFX - those mods need patches for this mod. It should load after Immersive Citizens, though.

It will have minor conflicts with mods such as Open Cities; those are generally harmless / can easily be patched by mod users in xEdit since it's so heavily dependent on load order.

Reply:
If it should be after Immersive Citizens then, 
put it in the same group as ICAIO, and add a load after as well.
 group: *lowPriorityGroup